node/src/components/consensus/highway_core/state/tests.rs

```diff
@@ -214,7 +214,7 @@ fn add_unit() -> Result<(), AddUnitError<TestContext>> {
     let b2 = add_unit!(state, rng, BOB, None; F, b1, c0)?;
 
     // The state's own panorama has been updated correctly.
-    assert_eq!(state.panorama, panorama!(F, b2, c1));
+    assert_eq!(state.panorama().clone(), panorama!(F, b2, c1));
```

Comment:
```suggestion
    assert_eq!(*state.panorama(), panorama!(F, b2, c1));
```
That should work, too, I think, without copying. If it doesn't:
```suggestion
    assert_eq!(state.panorama(), &panorama!(F, b2, c1));
```

Reply:
Done in https://github.com/CasperLabs/casper-node/pull/550/commits/dbc3dbbf6d706de6052bd675f2fe6b51088e81dd.